<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FFWaveDefinitionWidget</class>
 <widget class="QWidget" name="FFWaveDefinitionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldUsesDropship">
     <property name="text">
      <string>Uses Dropships</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="fieldSquadCount">
     <property name="toolTip">
      <string>This doesn't necessarily affect the number of squad types that the game engine will actually use for this round; it just controls whether certain squad types</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> squads</string>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelSquadCount">
     <property name="text">
      <string>Squad Count (apparently unused)</string>
     </property>
     <property name="buddy">
      <cstring>fieldSquadCount</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="fieldSquadOrder">
     <item>
      <property name="text">
       <string>Random</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ordered</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelSquadOrder">
     <property name="text">
      <string>Squad Order</string>
     </property>
     <property name="buddy">
      <cstring>fieldSquadOrder</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
